schematools/utils.py

```diff
@@ -117,14 +118,28 @@ def schema_fetch_url_file(schema_url_file):
 
 
 @lru_cache(maxsize=500)  # type: ignore
-def toCamelCase(name):
+def toCamelCase(name: str) -> str:
     """
     Unify field/column/dataset name from Space separated/Snake Case/Camel case
     to camelCase.
     """
-    name = " ".join(name.split("_"))
-    words = RE_CAMEL_CASE.sub(r" \1", name).strip().lower().split(" ")
-    return "".join(w.lower() if i == 0 else w.title() for i, w in enumerate(words))
+    out = StringIO()
+    after_space = False
+    first = True
```

Comment:
`first` is not really needed:
```
after_space = True
for c in name.strip(" _"):
     ....
```
Will have same effect.

Reply:
That's assuming spaces are all ASCII space (not newline, tab, NBSP). Can we make that assumption?

Reply:
Nope. Valid point.

Reply:
Also testNameMagic fails without first.

Reply:
I admit, suggestion was not really good. I am just concerned that new version becomes not much more readable than previous ;)
Except it is not using horrible regex, which is plus, unless you are Perl developer.